app/components/spotlight/select_image_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render Blacklight::System::ModalComponent.new do |component| %>
+  <% component.with_title { t('spotlight.pages.form.select_image') } %>
+  <% component.with_body do %>
+    <div class='p-3 m-2'>
+      <input type="hidden" name="select_image_region">
+      <p class="instructions"><%= help_text.html_safe %></p>
+      <div class="mx-3" id="resource_block_iiif_cropper" data-behavior="iiif-cropper" data-cropper-key="select_image_<%=@block_item_id%>" data-index-id="<%=@index_id%>" data-crop-width="120" data-crop-height="120">
+    </div>
+  <% end %>
+   <% component.with_footer do %>
+    <div> 
+      <input type='button' class='btn btn-outline-primary' value='Cancel' data-bl-dismiss='modal'>
+      <input id='saveimage' type='button' class='btn btn-primary' value='Save changes' data-bl-dismiss='modal'>  
```

Comment:
Maybe a nit? Does this need to be an ID? I think you use data-behavior someplace else to target your element. In the handler you have `'#blacklight-modal input#saveimage` which sort of implies you don't trust your own ID. If it needs to be an ID, I think something more specific/less prone to accidental collision would be nice.

Reply:
Changed this with input from @dnoneill and @taylor-steve

Reply:
But it still does use and id. That said, I think we were fine with it for the time being. Also I tried to make the id more understandable "save-cropping-selection"

Reply:
Thanks!